test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -727,6 +782,118 @@ public void CreateDisplayMetadata_IsFlagsEnum_ReflectsModelType(Type type, bool
                 .ThenBy(item => item.Key.Name, StringComparer.Ordinal));
         }
 
+        private DataAnnotationsMetadataProvider CreateLocalizingProvider()
```

Comment:
Not looking at other new test names 'til you check them throughout this PR. `ResourceType` is a supported way to localize `string`s in the system.